### PR TITLE
Enable multiclass classification with plsdaCaret

### PR DESCRIPTION
**Pull request by GegznaV**
Solves #2620.

Locally the new tests pass:
image

### CODE REVIEW

File: tests/testthat/test_classif_plsdaCaret.R [hunk 1]
```diff
@@ -36,3 +36,41 @@ test_that("classif_plsdaCaret", {
   testProbParsets("classif.plsdaCaret", binaryclass.df, binaryclass.target, binaryclass.train.inds,
     old.probs.list, parset.list)
 })
+
```
Comment (pat-s) [inline tests/testthat/test_classif_plsdaCaret.R]: Please remove the unnecessary empty lines below.

File: tests/testthat/test_classif_plsdaCaret.R [hunk 2]
```diff
@@ -36,3 +36,41 @@ test_that("classif_plsdaCaret", {
...
+    p2 = predict(m, newdata = newx, type = "prob")[, , 1]
+    old.predicts.list[[i]] = p
+    old.probs.list[[i]] = p2
+  }
+
+  testSimpleParsets("classif.plsdaCaret", multiclass.df, multiclass.target, multiclass.train.inds,
+    old.predicts.list, parset.list)
+  testProbParsets("classif.plsdaCaret", multiclass.df, multiclass.target, multiclass.train.inds,
+    old.probs.list, parset.list)
+
```
Comment (pat-s) [inline tests/testthat/test_classif_plsdaCaret.R]: This line is also not not needed.

Comment (GegznaV) [reply tests/testthat/test_classif_plsdaCaret.R]: Removed two lines.

File: tests/testthat/test_classif_plsdaCaret.R (hunk 1 above)
Comment (GegznaV) [reply tests/testthat/test_classif_plsdaCaret.R]: Removed one line.

**Comment by GegznaV** [comment]
I removed 3 lines and applied `mlr_style`. Then added a line to NEWS.md. 
Still, I'm not sure if one more line should be removed?

**Comment by pat-s** [comment]
Thanks for contributing!

**Pull request merged**